src/components/position-at-box-shape-border.js

```diff
@@ -59,6 +59,8 @@ AFRAME.registerComponent("position-at-box-shape-border", {
   tick() {
     if (!this.target) {
       this.targetEl = this.el.querySelector(this.data.target);
+      if (!this.targetEl) return;
```

Comment:
Hmm, I assume this is to deal with async targets, but it feels wrong that we would just be silently ignoring this. Combined with my above comment about running querySelector in tick, seems like the contract should be that its up to whatever is adding the `position-at-box-shape-border` to ensure that target exists before adding this component, then the `targetEl` can just be resolved in `init`

Reply:
these are mostly in `hub.html`, i'll see if i can diagnose what caused this to start breaking for this specific feature

Reply:
I couldn't repro this, might have been an issue on my home machine. I'll add a warning and revisit later if I see it hitting again.